Game/3d-game-project/3D/Maps/Forest/house.gd

```diff
@@ -0,0 +1,20 @@
+extends Node3D
+class_name House
+
+@onready var main = NodePath("/root/GameController")
+
+func _on_interactable_focused(interactor: Interactor) -> void:
+	pass # Replace with function body.
```

Comment:
"Replace with a function body" should. be remove or the said function should be added